cmd/nigiri/resources/docker-compose.yml

```diff
@@ -16,7 +15,7 @@ services:
       - 28332:28332
       - 28333:28333
     volumes:
-      - ./volumes/bitcoin:/root/.bitcoin
+      - ./volumes/bitcoin:/data/.bitcoin
```

Comment:
Weird you need the path to be changed, it was working till now. Care to elaborate?